### PR TITLE
add article about profiling and view dependencies

### PR DESCRIPTION
**Pull request by exaSR**
simple enough.
The preformatted text blocks might be too wide for Salesforce?

Using our CDP database as example, I'm not aware of a public (Exasol) data model with the required view complexity.

### CODE REVIEW

File: Data-Science/how-to-find-problem-views-from-profiling.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+As is often the case with dependency tracking, we require a top-level view to start our search.
+In the following **example**, we will use the view `CUSTOMER_STATS_RLS.F_MONITOR` as entry point.
+
+Selecting from the view will provide a profile which includes the following pipeline:
+
+|PART_ID|PART_NAME|PART_INFO|OBJECT_SCHEMA|OBJECT_NAME|
+|---|---|---|---|---|
+|7|PIPE SCAN|on REPLICATED table|CDP|DATABASE_JI|
+|8|PIPE JOIN|(null)|CDP|ISSUES|
```
Comment (allipatev) [inline Data-Science/how-to-find-problem-views-from-profiling.md:23]:
```suggestion
|8|PIPE JOIN||CDP|ISSUES|
```

File: Data-Science/how-to-find-problem-views-from-profiling.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+|9|PIPE JOIN|on REPLICATED table|CDP|ISSUESTATI|
+|10|PIPE FILTER|(null)|(null)|(null)|
```
Comment (allipatev) [inline Data-Science/how-to-find-problem-views-from-profiling.md:25]:
```suggestion
|10|PIPE FILTER||||
```

File: Data-Science/how-to-find-problem-views-from-profiling.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+|11|PIPE INDEX OUTER JOIN|on REPLICATED table|CDP|ISSUEVERSIONS|
+|12|PIPE PREFERENCE PREFILTER|(null)|(null)|(null)|
```
Comment (allipatev) [inline Data-Science/how-to-find-problem-views-from-profiling.md:27]:
```suggestion
|12|PIPE PREFERENCE PREFILTER||||
```

File: Data-Science/how-to-find-problem-views-from-profiling.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+### Analysis Query
+
+We will use the following query (template) to perform dependency analysis:
+```sql
+with k1 as (
+    select
+        sys_connect_by_path(object_schema||'.'||object_name, ' > ') as pth,
+        referenced_object_schema,
+        referenced_object_name
+    from sys.exa_all_dependencies
```
Comment (allipatev) [inline Data-Science/how-to-find-problem-views-from-profiling.md:44]: Maybe [EXA_DBA_DEPENDENCIES​_RECURSIVE](https://docs.exasol.com/db/latest/sql_references/system_tables/metadata/exa_dba_dependencies_recursive.htm) could be useful here, to hide CONNECT BY complexity.

File: Data-Science/how-to-find-problem-views-from-profiling.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+Let's say we are interested how `ISSUES` is used twice, and how it relates to `DATABASE_JI`.
+
+### Analysis Query
+
+We will use the following query (template) to perform dependency analysis:
+```sql
+with k1 as (
+    select
+        sys_connect_by_path(object_schema||'.'||object_name, ' > ') as pth,
```
Comment (allipatev) [inline Data-Science/how-to-find-problem-views-from-profiling.md:41]:
```suggestion
        sys_connect_by_path('"'||object_schema||'"."'||object_name||'"', ' > ') as pth,
```

File: Data-Science/how-to-find-problem-views-from-profiling.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+- The example above does not contain any actual problems. It was simply picked to illustrate the process.
+- While the example is built to track down *TABLE* usage, the analysis query will also work for *VIEWS*, *FUNCTIONS* and
+  (non-procedure) *SCRIPTS*.
+
+## Additional References
+
+1. User Manual: [Profiling](https://docs.exasol.com/db/latest/database_concepts/profiling.htm)
+1. User Manual: [EXA_DBA_DEPENDENCIES](https://docs.exasol.com/db/latest/sql_references/system_tables/metadata/exa_dba_dependencies.htm)
+
+
```
Comment (allipatev) [inline Data-Science/how-to-find-problem-views-from-profiling.md:121]:
```suggestion
*We appreciate your input! Share your knowledge by contributing to the Knowledge Base directly in [GitHub](https://github.com/exasol/public-knowledgebase).* 
```

**Review by allipatev — APPROVED** [review]
- "(null)" removal - my personal preference. Disputable.
- Reliable handling of delimited identifiers looks relatively important to me. Though if you agree, it might be good to adapt accordingly the further dependent examples.
- Footer is taken from [Templates](https://github.com/exasol/public-knowledgebase/tree/main?tab=readme-ov-file#templates).
- EXA_DBA_DEPENDENCIES​_RECURSIVE - just as an idea.

File: Data-Science/how-to-find-problem-views-from-profiling.md (hunk 4 above)
Comment (exaSR) [reply Data-Science/how-to-find-problem-views-from-profiling.md:44]: Unfortunately, the recursive table is useless if you want to find the full path leading from source to target.

File: Data-Science/how-to-find-problem-views-from-profiling.md (hunk 5 above)
Comment (exaSR) [reply Data-Science/how-to-find-problem-views-from-profiling.md:41]: Let's leave that part to the user, as
(1) The names will have to be copy-pasted as String anyway (filter for system tables)
(2) Adding another `replace(identifier, '"', '""')` will not make it more readable.

**Pull request merged**